pull_api.md

```diff
@@ -0,0 +1,302 @@
+        <country>Deutschland</country>
+      </job_offer_location>
+      <job_offer_location>
+        <city>Berlin</city>
+        <street>Oranienburger Straße 33</street>
+        <zip>10178</zip>
+        <country>Deutschland</country>
+      </job_offer_location>
+    </job_offer_locations>
+
+    <description_headline>Komm' an Bord!</description_headline>
+    <description>
+      <![CDATA[
+        <p>
+          Unser renormiertes Seefahrtsunternehmen heuert verlässlich und regelmäßig die besten
+          Seefrauen und -männer (w/m/d) östlich des Atlantiks an. Generationen von Seefahrtsexperten
+          haben ihre Karriere mit ersten nautischen Schritten in unserem Unternehmen gestartet.
+        </p>
+      ]]>
+    </description>
+
+    <tasks_headline>Du kannst im Schlaf unter tosendem Sturm problemlos Segel setzen?</tasks_headline>
+    <tasks>
+      <![CDATA[
+        <ul>
+          <li>Du peppst die rauhen und kantigen Geschichten unserer weltweit verstreuten Mitarbeiter linguistisch gekonnt auf</li>
+          <li>Dein Instikt lässt dich spielerisch prüfbare Fakten von hanebüchenem Seemannsgarn unterscheiden</li>
+        </ul>
+      ]]>
+    </tasks>
+
+    <qualifications_headline>Du bist mutig genug, in allen Ozeanen dieser Welt zu schwimmen?</qualifications_headline>
+    <qualifications>
+      <![CDATA[
+        <ul>
+          <li>Du kannst schwimmen (Seepferdchen notwendig)</li>
+          <li>Ein leichtes Schaukeln ist für dich kein Problem</li>
+        </ul>
+      ]]>
+    </qualifications>
+
+    <benefits_headline>Bestens ausgerüstet!</benefits_headline>
+    <benefits>
+      <![CDATA[
+        <ul>
+          <li>Reise zum Ende der Welt</li>
+          <li>Ein loyales Team</li>
+          <li>Remote first: Wir begrüßen ausdrücklich das eigenständige, asynchrone Arbeiten in entlegenen Gegenden</li>
+        </ul>
+      ]]>
+    </benefits>
+
+    <contact_headline>Frag' unsere Expertin Elaine!</contact_headline>
+    <contact>
+      <![CDATA[
+        <p>Elaine Marley</p>
+        <p>elaine.marley@absolventa.de</p>
+        <small>Senior Seefahrt-Personalmanagerin</small>
+      ]]>
+    </contact>
+
+    <company_description_headline>Unser Geschäft ist kein Voodoo!</company_description_headline>
+    <company_description>
+      <![CDATA[
+        <p>Marley & Threepwood Ventures - Ihr Partner in Sachen Seefahrt- und Handelsbeziehungen</p>
+      ]]>
+    </company_description>
+
+    <video_url><![CDATA[https://api.example.com/videos/seemannsgarn]]></video_url>
+    <header_image_url><![CDATA[https://api.example.com/content/header_image.jpg]]></header_image_url>
+    <color>#efefef</color>
+    <custom_company_title>Threepwood GmbH</custom_company_title>
+
+    <!--attributes only relevant for azubi.de:  -->
+    <apprenticeship_started_at type='datetime'>
+      2017-07-07T00:00:00+02:00
+    </apprenticeship_started_at>
+    <minimal_degree>basic</minimal_degree>
+    <duration>36</duration>
```

Comment:
Maybe just for clarification we can add an comment about the end of the azubi specific attributes?
<!-- end of azubi.de specific attributes  -->

Reply:
Good point, I added a second comment.